server/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5", /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
+    "target": "es6", /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
```

Comment:
I think this should be left as `es5`. This option is simply to tell TS what version of JS it wants to compile to. We can code in es6+ regardless, but having this set as `es5` allows the compiled JS to be backwards compatible.

Reply:
Sequelize seems to break if I switch this to es5. I will see if I can fix this

Reply:
Something broke while I was testing it, that's why I switched to ES6. I will switch it back to ES5 since it seems to work fine now.

Reply:
Changing back to es5 still? Or are we keeping it?

Reply:
We will do ES6, since that is what we switched to in the client directory and what Daniel did on his PR